lib/modules/datasource/github-tags/index.spec.ts

```diff
@@ -122,6 +122,10 @@ describe('modules/datasource/github-tags/index', () => {
           hash: 'abc',
         },
       ]);
+      jest.spyOn(githubGraphql, 'queryReleases').mockResolvedValueOnce([
+        { version: 'v1.0.0', releaseTimestamp: '2021-01-01', isStable: true },
+        { version: 'v2.0.0', releaseTimestamp: '2022-01-01', isStable: false },
+      ] as never);
```

Comment:
can we use partial instead otherwise typescript can't detect required type changes